### PR TITLE
Tech

### PR DESCRIPTION
**Pull request by leonardofrp5**
**Summary**

Design of the technologies card

**Trello Card**

https://trello.com/c/fTCcb335/22-tecnolog%C3%ADa

**Screenshots**

captura de pantalla 2018-11-01 a la s 7 58 59

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -9,6 +9,16 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <h1 class="title">Front-End Tools</h1>
+    <div class="card">
```
Comment (henryzarza) [inline maquetado/index.html]: This must be at the same level of `h1`

File: maquetado/index.html [hunk 2]
```diff
@@ -9,6 +9,16 @@
...
+      <div class="tech">
+        <img class="logo" src="assets/react_icon.svg">
```
Comment (henryzarza) [inline maquetado/index.html]: I recommend you to set the attribute `alt` and change the class name for something more descriptive, for example: `tech-logo`

File: maquetado/index.html [hunk 3]
```diff
@@ -9,6 +9,16 @@
...
+        <h4>ReactJS</h4>
+        <h4 class="title-bold">Release: 2013</h4>
```
Comment (henryzarza) [inline maquetado/index.html]: I think that you should change this for `h5`

File: maquetado/index.html [hunk 4]
```diff
@@ -9,6 +9,16 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <h1 class="title">Front-End Tools</h1>
+    <div class="card">
+      <div class="tech">
```
Comment (henryzarza) [inline maquetado/index.html]: I recommend you to keep the structure as clean as posible, please change it

File: maquetado/scss/_tech.scss [hunk 5]
```diff
@@ -0,0 +1,40 @@
+.card {
+  background: $green;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: Change this for `background-color`

File: maquetado/scss/_tech.scss [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  border-radius: 5px;
+  height: 270px;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: Remember to use `max-height` property. Check this [style guide](https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md#setting-a-fixed-width)

File: maquetado/scss/_tech.scss [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  width: 180px;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss:10]: Change this value for `200px`

File: maquetado/scss/_tech.scss [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+  &::before {
+    border: solid 1px $green;
+    border-radius: 5px;
+    content: '';
+    height: 270px;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: Change this value for `100%`

File: maquetado/scss/_tech.scss [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+    margin: -7px;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: This isn't necessary so delete it. You can use `left` and `top` properties

File: maquetado/scss/_tech.scss [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+    position: absolute;
+    width: 180px;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: Change this value for `100%`

File: maquetado/scss/_tech.scss [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+  }
+}
+
+.tech {
+  position: relative;
+  h4 {
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: Remember that isn't a good practice this nesting and to set styles a general elements

File: maquetado/scss/_tech.scss [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+    border-radius: 5px;
+    content: '';
+    height: 270px;
+    margin: -7px;
+    position: absolute;
+    width: 180px;
+  }
+}
+
+.tech {
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: I think that this styles should be in `.card`

File: maquetado/scss/_tech.scss [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+  position: relative;
+  h4 {
+    display: flex;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: You don't need this

File: maquetado/scss/_tech.scss [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+    justify-content: center;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: The same here

File: maquetado/scss/_tech.scss [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+    margin-top: 10px;
+  }
+  p {
+    font-size: $text-small;
+    padding: 10px 19px;
+    text-align: center;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: I think that you don't need this

File: maquetado/scss/_tech.scss [hunk 16]
```diff
@@ -0,0 +1,40 @@
...
+  }
+}
+.logo {
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: Add a break line

File: maquetado/scss/_tech.scss [hunk 17]
```diff
@@ -0,0 +1,40 @@
...
+  align-items: center;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: You don't need this

File: maquetado/scss/_tech.scss [hunk 18]
```diff
@@ -0,0 +1,40 @@
...
+  display: flex;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: The same here

File: maquetado/scss/_colors.scss [hunk 19]
```diff
@@ -0,0 +1,3 @@
+$blue: #00ADEE;
```
Comment (henryzarza) [inline maquetado/scss/_colors.scss]: Remember the structure of the folders that i tell you. `scss/variables/_colors.scss`

File: maquetado/scss/_tech.scss [hunk 20]
```diff
@@ -0,0 +1,40 @@
...
+  height: $tech-logo-size;
+  margin: auto;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: You don't need this

File: maquetado/scss/_tech.scss [hunk 21]
```diff
@@ -0,0 +1,40 @@
...
+  padding: 11px;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: The same here

File: maquetado/scss/_tech.scss [hunk 22]
```diff
@@ -0,0 +1,40 @@
...
+}
+.title-bold {
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss:37]: Add a break line

File: maquetado/scss/_variables.scss [hunk 23]
```diff
@@ -0,0 +1,6 @@
+$text-small: 12px;
```
Comment (henryzarza) [inline maquetado/scss/_variables.scss]: The same recommendation here with the structure

**Review by henryzarza — CHANGES_REQUESTED** [review]

**Review by henryzarza — APPROVED** [review]

File: maquetado/index.html [hunk 24]
```diff
@@ -9,6 +9,15 @@
...
+  <h1 class="title">Front-End Tools</h1>
+  <div class="tech">
+    <img class="tech-logo" alt="React logo" src="assets/react_icon.svg">
+    <h4 class="tech-title">ReactJS</h4>
+    <h5 class="tech-title title-bold">Release: 2013</h5>
+    <p class="tech-content">
+      React is declarative, component-based JavaScript library for bulding user interfaces
+    </p>
+  </div>
+</div>
```
Comment (henryzarza) [inline maquetado/index.html]: Delete this, isn't necessary

File: maquetado/scss/_tech.scss [hunk 25]
```diff
@@ -0,0 +1,39 @@
+.tech {
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  max-width: 100%;
+  padding: 14px;
+  position: relative;
+  text-align: center;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss:9]: Move this to `.tech-content`class style definition

**Review by henryzarza — CHANGES_REQUESTED** [review]
Good job! The last changes and go

File: maquetado/scss/_tech.scss (hunk 25 above)
Comment (leonardofrp5) [reply maquetado/scss/_tech.scss:9]: This attribute I didn't move because is necessary for the titles and content

**Review by henryzarza — APPROVED** [review]
Remember change the `width` of the card

**Pull request merged**